tests/lean/traceTacticSteps.lean.expected.out

```diff
@@ -0,0 +1,44 @@
+[Elab.step] expected type: Sort ?u, term
+True
+[Elab.step] expected type: True, term
+by 
+  skip 
+  trivial
+[Elab.step] 
+  skip 
+  trivial
+[Elab.step] 
+  skip 
+  trivial
+[Elab.step] skip
+[Elab.step] trivial
+[Elab.step] (apply And.intro✝) <;> trivial
+[Elab.step] focus 
+  (
```

Comment:
Hmm, interesting formatting!

Reply:
Both newlines are because tacticSeq1Indented prints a line before every tactic.  We have quite a few interesting formatting corner cases like that:
```lean
def foo :=
  do
    putStrLn "hello world"

theorem bar :=
  by
    rfl

theorem baz :=
  by
    ·
      rfl
```

You can browse through the [mathport output](https://github.com/leanprover-community/mathlib3port/tree/master/Mathbin) to find more of these artworks.